app/database_etl/test_adjustment_handler/test_adjustment.py

```diff
@@ -24,6 +24,15 @@ def logit(p, tol = 1e-3):
     # return the logit of the constrained probability
     return log(p / (1 - p))
 
+def result_is_bounded(median_adj_prev, raw_prev, maxsmall = 0.5, minbig = 0.5):
+    logit_delta = logit(median_adj_prev) - logit(raw_prev)
+    adjusted_closeto_raw = abs(logit_delta) < 1
+    
+    both_below_maxsmall = (median_adj_prev <= maxsmall) and (raw_prev <= maxsmall)
```

Comment:
Can you explain the significance behind these two variables here

Reply:
Removed `maxsmall` and `minbig` -- these are better for fixed values. Added comments to explain.